Podfile

```diff
@@ -3,4 +3,12 @@ use_frameworks!
 target 'MCSCommerceWeb' do
   pod 'SVGKit', :git => 'https://github.com/SVGKit/SVGKit.git', :branch => '3.x'
 
-end
\ No newline at end of file
+  pod "CocoaLumberjack"	
```

Comment:
Remove CocoaLumberjack, we're already using DXPLogger as dependency

Reply:
removed